pkg/pgclient/client.go

```diff
@@ -23,8 +23,8 @@ import (
 
 // Config for the database
 type Config struct {
-	host                    string
-	port                    int
+	Host                    string
```

Comment:
nit: I would expose  Config.TcpAddr() (string) instead of making these public. but will leave up to you.